File: Modules/Sources/WordPressKitObjC/RemoteReaderCrossPostMeta.m
#import "RemoteReaderCrossPostMeta.h"

@implementation RemoteReaderCrossPostMeta

- (instancetype)init {
    self = [super init];
    if (self) {
        _postID = @0;
        _siteID = @0;
        _siteURL = @"";
        _postURL = @"";
        _commentURL = @"";
    }
    return self;
}

@end

File: WordPressKit/WordPressKit/RemoteReaderCrossPostMeta.swift
import Foundation

open class RemoteReaderCrossPostMeta: NSObject {
    @objc open var postID: NSNumber = 0
    @objc open var siteID: NSNumber = 0
    @objc open var siteURL = ""
    @objc open var postURL = ""
    @objc open var commentURL = ""
}
